src/components/atom/AppCalendar.tsx

```diff
@@ -0,0 +1,52 @@
+import { Calendar, CalendarChangeParams, CalendarMonthNavigatorTemplateParams } from "primereact/calendar"
+import { useTranslation } from "react-i18next";
+import { namespaces } from "../../i18n/i18n.constants";
+import { Dropdown, DropdownChangeParams } from "primereact/dropdown";
+import 'primereact/resources/primereact.css';
+import 'primeicons/primeicons.css';
+import { current } from "@reduxjs/toolkit";
+
+interface AppCalendarProps {
+    id?: string
+    placeholder?: string,
+    selectedDate?: Date,
+    setDate: (date: string) => void
+    yearRange?: string
+}
+
+const currentYear = new Date().getFullYear()
+const MIN_BIRTH_YEAR = currentYear - 120
+const MAX_BIRTH_YEAR = currentYear
+const _yearRange = `${MIN_BIRTH_YEAR}:${MAX_BIRTH_YEAR}`
+const years = Array.from({length: MAX_BIRTH_YEAR-MIN_BIRTH_YEAR + 1}, (_, i) => i + MIN_BIRTH_YEAR)
+
+const AppCalendar = ({id = 'datePicker', placeholder = 'Fecha de nacimiento', selectedDate, setDate, yearRange}: AppCalendarProps): JSX.Element => {
+    const {t: translate} = useTranslation(namespaces.atoms.appCalendar);
```

Comment:
this one is obvious, but it some other places it is not worth saving a couple letters when naming variables as it makes it less readable.